docs/getting-started/installation.md

```diff
@@ -103,6 +103,18 @@ colcon build --packages-up-to beluga_example --cmake-args -DBUILD_TESTING=OFF
 
 ::::
 
+After building, don't forget to source the environment:
+
+::::{tab-set}
+
+:::{tab-item} Ubuntu
+```bash
+source install/setup.bash
+```
+:::
+
+::::
```

Comment:
@DPR00 nit: consider moving this fragment to the end of the page, as it applies to ROS 1 and ROS 2 alike.

Reply:
Yes, I've just modified it.